probes/contributorsWithOrgOrCompany/def.yml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: contributorsWithOrgOrCompany
+short: Checks whether a project has a contributions from Github users associated with a company or organization.
+motivation: >
+  
+implementation: >
+  This probe looks for the the companies and organizations that contributors of the project belong to. 
+outcome:
+  - If the project has contributions from Github users belonging to an organization or company, then the probe returns OutcomePositive.
+remediation:
+  effort: Low
```

Comment:
is remediation low or high? In the sense that it's fairly hard to remediate, I think. Wdut?

Reply:
Ah yes, I agree.